Microsoft.Azure.Cosmos.Encryption/src/EncryptionContainer.cs

```diff
@@ -45,6 +56,10 @@ internal sealed class EncryptionContainer : Container
 
         public override Database Database => this.container.Database;
 
+        public Dictionary<List<string>, string> ToEncrypt = new Dictionary<List<string>, string>();
+
+        List<EncryptionOptions> pEncryptionOptions = null;
```

Comment:
Then it can be moved up with other private members.
```suggestion
        private readonly IReadOnlyList<EncryptionOptions> encryptionOptions;
```